src/Button.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+
+const Button =
+({type, className, clickHandler, name}) => {
+    return (
+        <button
+            name={name}
+            type={type}
+            className={className}
+            onClick={clickHandler}
+        >
+        </button>
+    );
+}
```

Comment:
Dit is nu meer een doorgeefluik. Ten opzichte van het html button element zelf gebeurd er verder niets in dit component. Je had dus net zo goed direct kunnen gebruiken